spec.md

```diff
@@ -147,10 +147,40 @@ A web-friendly display name is camel-cased with spaces and punctuation supported
 | metadata | JSON object | An opaque object of metadata for a service plan. Controller treats this as a blob. Note that there are [conventions](https://docs.cloudfoundry.org/services/catalog-metadata.html) in existing brokers and controllers for fields that aid in the display of catalog data. |
 | free | boolean | When false, instances of this plan have a cost. The default is true. |
 | bindable | boolean | Specifies whether instances of the service plan can be bound to applications. This field is OPTIONAL. If specified, this takes precedence over the `bindable` attribute of the service. If not specified, the default is derived from the service. |
+| [schemas](#schema-object) | object | Schema definitions for service instances and bindings for the plan. |
 
 
 \* Fields with an asterisk are REQUIRED.
 
+##### Schema Object
+
+| Response field | Type | Description |
+| --- | --- | --- |
+| [service_instance](#service-instances-object) | object | The schema definitions for all actions for a service instance. |
+| [service_binding](#service-bindings-object) | object | The schema definitions for all actions for a service binding. Used only if the service is bindable. |
+
+
+##### Service Instances Object
+
+| Response field | Type | Description |
+| --- | --- | --- |
+| [create](#input-parameters-object) | object | The schema definitions for the input parameters. |
+| [update](#input-parameters-object) | object | The schema definitions for the input parameters. |
+
+
+##### Service Bindings Object
+
+| Response field | Type | Description |
+| --- | --- | --- |
+| [create](#input-parameters-object) | object | The schema definitions for the input parameters. |
+
+
+##### Input Parameters Object
+
+| Response field | Type | Description |
+| --- | --- | --- |
+| parameters | JSON schema object | The schema definitions for the input parameters. Schema definitions MUST be valid [JSON Schema draft v4](http://json-schema.org/). It is recommended for broker authors explicitly mention the JSON schema draft 4 version, as this will prevent breaking changes when in the future the OSB spec starts supporting future versions of JSON schema with low readability impact. Each input parameter is expressed as a property within a JSON object. A service supporting no input parameters SHOULD provide the following schema: `{ "$schema": "http://json-schema.org/draft-04/schema#", "additionalProperties": false }`. |
```

Comment:
It's up to the platforms. In CF world, the cloud controller _probably will_ perform validation, as will client tools that are using this (web UIs and CLIs). There don't seem to many times in the spec where we specify what platforms should/must/may do.